susemanager-utils/testing/docker/scripts/push-to-obs.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh -e
```

Comment:
I am not really sure if this script should go here or at `rel-eng`

Reply:
This script will be called to submit and it uses the container, right? In this case it has nothing to do with the test.
I think moving this to re-eng is a good idea,

Reply:
hmm, no. this is the script which is called inside of the conatainer? Then it belongs here.
More a question about the one in testing/automation to move to rel-eng